test/framework/clusterctl/client.go

```diff
@@ -230,7 +232,18 @@ func ConfigCluster(ctx context.Context, input ConfigClusterInput) []byte {
 		TargetNamespace:          input.Namespace,
 	}
 
-	clusterctlClient, log := getClusterctlClientWithLogger(input.ClusterctlConfigPath, fmt.Sprintf("%s-cluster-template.yaml", input.ClusterName), input.LogFolder)
+	options := []clusterctlclient.Option{}
+	if input.VariableOverwrites != nil {
+		configClient, err := config.New(input.ClusterctlConfigPath)
```

Comment:
Q: This pattern is fine, but what do you think about using the same pattern as above? (with `copyAndAmendClusterctlConfig`). For consistency and also makes it easier to debug to have a separate clusterctl config.

(I might be missing something if it's not possible)